x/concentrated-liquidity/position.go

```diff
@@ -296,20 +296,20 @@ func (k Keeper) deletePosition(ctx sdk.Context,
 	}
 	store.Delete(positionIdKey)
 
-	// Remove the address-pool-position ID to position mapping.
-	addressPoolIdPositionIdKey := types.KeyAddressPoolIdPositionId(owner, poolId, positionId)
-	if !store.Has(addressPoolIdPositionIdKey) {
-		return types.AddressPoolPositionIdNotFoundError{Owner: owner.String(), PoolId: poolId, PositionId: positionId}
-	}
-	store.Delete(addressPoolIdPositionIdKey)
-
 	// Remove the pool-position ID mapping.
 	poolIdKey := types.KeyPoolPositionPositionId(poolId, positionId)
 	if !store.Has(poolIdKey) {
 		return types.PoolPositionIdNotFoundError{PoolId: poolId, PositionId: positionId}
 	}
 	store.Delete(poolIdKey)
 
+	// Remove the address-pool-position ID to position mapping.
```

Comment:
note: wondering if the order of deletion matters?  because i had to move this down because, we would never hit `PoolPositionIdNotFoundError` case because the `AddressPoolPositionIdNotFoundError` would always error first. 

i can move it back up but it'll be a slightly more complex test.

Reply:
It doesn't. If error occurs, any state updates are reverted in `runTx` in base app